public/views/walletHome.html

```diff
@@ -219,7 +216,17 @@ <h4 class="title m0">
          </div>
         </div>
 
-        <div ng-repeat="btx in index.txHistory"
+        <div id="searchLabel" class="left-inner-addon ">
+          <i class="fi-magnifying-glass"></i>
+          <input name="search"
+                ng-click="home.searchInput()"
+                ng-focus="home.formFocus('search')"
```

Comment:
When I focus the input the menu bar hides. And I use the ng-blur to show the menu bar when the input has lost focus.